### ISSUE TITLE
Query string URL paths ignored

### ISSUE THREAD

**Issue by manafire** [harlan-zw/unlighthouse#207]
### Describe the bug

I'm trying to crawl an old WordPress site where every page/post is reference via query param.  I tried disabling dynamicSampling and even manually specifying the URLs but they don't come up in the report.

```js
  urls: [
    '/', 
    '/?page_id=1193',
    '/?page_id=43',
    '/?page_id=856',
  ],  
```

=> Unlighthouse has finished scanning .... 1 routes in 15s.

**Comment by ErikCarlson-RGA** [comment]
I can confirm this ?unintended? functionality. We have pages that load different assets and content based on the ?id=num values. It is important to be able to use query strings with the ever growing nature if single page reactive sites.

**Comment by harlan-zw** [comment]
Hi, thanks for the ping on this issue, i've pushed up a fix in [v0.14.0](https://github.com/harlan-zw/unlighthouse/releases/tag/v0.14.0).

**Comment by vstoimenov10** [comment]
@mathiasrw thank you! So (un)lighthouse should be able to handle the URLs? I am trying to use it but it only scans the login page (the only URL without #). For all other URL's I list, it redirects back to the login URL. Unfortunately the site I am trying to scan is developed by a contractor, so I have very limited visibility. I have to do the scans on local host and don't have access to the project itself.
